server/services/location.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios';
+import GOOGLE_API_KEY from '../config/development';
```

Comment:
Was this meant to be a default import? It seems a bit strange that the default export from this config file would be just a single API key.